modules/tinymce/src/core/main/ts/api/dom/RangeUtils.ts

```diff
@@ -1,5 +1,7 @@
 import { Fun } from '@ephox/katamari';
 
+import { expandRng } from 'tinymce/core/fmt/ExpandRange';
```

Comment:
Forgot to leave a comment on this sorry... this isn't how we normally do imports either. There's 2 issues here:
1. This is an alias import which should never be done inside a `main` directory (the linter is failing on this)
2. We prefer `import * as` imports to it's clear where the function is coming from.

So with that in mind this should be something like `import * as ExpandRange from '../../fmt/ExpandRange'`.